### PR TITLE
fix(action-bar): removed thick border around action-bar when nested in the shell-center-row

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #3494

## Summary

Removed unnecessary border from action bar inside shell center row.

action-bar-border

### CODE REVIEW

**Comment by Elijbet** [comment]
@macandcheese  Removing the thick border on top, left, and bottom reveals that article header does not line up with action bar.

**Review by driskull — APPROVED** [review]
👍 Nice @Elijbet

File: src/components/calcite-shell-center-row/calcite-shell-center-row.scss [hunk 1]
```diff
@@ -56,8 +56,8 @@
 }
 
 ::slotted(calcite-action-bar) {
-  @apply border-solid border-color-3;
-  border-inline-end-width: theme("borderWidth.default");
+  border-inline-end: 1px solid;
+  @apply border-color-3;
 }
 ::slotted(calcite-action-bar[position="end"]) {
   @apply border-solid border-color-3;
```
Comment (asangma) [inline src/components/calcite-shell-center-row/calcite-shell-center-row.scss:63]: You'll need to do a similar style for `position="end"`.

**Review by driskull — APPROVED** [review]
👍

**Pull request merged**